src/renderer/components/NewProjectModal/ImportMediaView.tsx

```diff
@@ -1,116 +1,136 @@
+    return null;
+  }
+
+  const projectName = currentProject.name;
+
   const setCurrentProject = (project: Project) =>
     dispatch(projectCreated(project));
-
   const addToRecentProjects = async (project: Project) => {
     const projectMetadata = await retrieveProjectMetadata(project);
     dispatch(recentProjectAdded({ ...project, ...projectMetadata }));
   };
 
-  const onProjectCreate = async () => {
-    setAwaitingProjectCreate(true);
+  const handleTranscribe = async () => {
+    const project = await updateProjectWithMedia(currentProject, mediaFilePath);
 
-    const project = await makeProject(projectName, mediaFilePath);
-    if (project === null) {
-      setAwaitingProjectCreate(false);
+    if (project === null || mediaFilePath === null) {
       return;
     }
 
     setCurrentProject(project);
     await addToRecentProjects(project);
+
     nextView();
   };
 
+  const transcribeButton = (
+    <CustomButton
+      color="primary"
+      onClick={handleTranscribe}
+      disabled={isAwaitingMedia}
+      sx={{ width: '40%' }}
+    >
+      Transcribe
+    </CustomButton>
+  );
+
+  const cancelButton = (
+    <CustomButton color="secondary" onClick={prevView} sx={{ width: '40%' }}>
+      Back
+    </CustomButton>
+  );
+
   return (
-    <>
-      <Typography fontWeight="bold">New Project</Typography>
-      <CustomTextField
-        sx={{ marginTop: '40px' }}
-        label="Project Name"
-        variant="outlined"
-        value={projectName}
-        onChange={(event) => setProjectName(event.target.value)}
-      />
-      <SelectMediaBlock
-        mediaFilePath={mediaFilePath}
-        setMediaFilePath={setMediaFilePath}
-      />
-      <ButtonContainer>
-        <CancelButton onClick={closeModal}>Cancel</CancelButton>
-        <ActionButton
-          onClick={onProjectCreate}
-          disabled={isAwaitingProjectCreate}
-        >
-          Create!
-        </ActionButton>
-      </ButtonContainer>
-    </>
+    <Container sx={{ height: { xs: 500 } }}>
+      <CustomColumnStack
+        alignItems="flex-start"
+        justifyContent="space-between"
+        sx={{ height: '50%' }}
+      >
+        <CustomRowStack justifyContent="space-between">
+          <Typography variant="h1" sx={{ color: colors.grey[400] }}>
+            {projectName}
+          </Typography>
+          <IconButton
+            sx={{ color: colors.yellow[500], fontSize: 20 }}
+            onClick={closeModal}
+          >
+            <CloseIcon />
+          </IconButton>
+        </CustomRowStack>
+        <CustomRowStack justifyContent="center">
+          <SelectMediaBlock
+            setMediaFileName={setMediaFileName}
+            setMediaFilePath={setMediaFilePath}
+            setIsAwaitingMedia={setIsAwaitingMedia}
+          />
+        </CustomRowStack>
+      </CustomColumnStack>
+      <CustomColumnStack
+        alignItems="baseline"
+        justifyContent="flex-start"
+        sx={{ height: '45%' }}
```

Comment:
Love the stacks. Flexbox convert. Looks like this stack (which displays the uploaded media) is a little bit big and its pushing the back/transcribe buttons down. Maybe reduce the height a bit. 

Being aware of multiple files in the future or even just ultra-long filenames now, could be a good idea to add overflow-y: 'auto' (overflowY ?) to this stack. This will add a scrollbar when the content overflows out of the designated height for the container element.

Reply:
Nice catch, I went ahead and adjusted the hieght such that it lines up. I also removed `marign-top` in other components.

Most importantly i also added the `overflowY: 'auto'` in this commit: https://github.com/patrickbrett/mlvet/pull/38/commits/eaf8056a93593b9f6945a1618fecd871dbe21670